interface.c

```diff
@@ -255,7 +256,7 @@ void game_start_stop(GtkMenuItem     *widget,
       from_virtual();
       move_block(0,0,0);
       current_level = options.level;
-      update_game_values(0,current_level,0);
+      update_game_values();
```

Comment:
Oh how did this work?

Reply:
The function is defined as `void update_game_values(void)`
It did compile with the extra params.. C allows crazy things.